c/common/src/codingstandards/c/UndefinedBehavior.qll

```diff
@@ -6,28 +6,53 @@ import codingstandards.cpp.UndefinedBehavior
  */
 abstract class CUndefinedBehavior extends UndefinedBehavior { }
 
+class PointerOrArrayType extends DerivedType {
```

Comment:
`PointerOrArrayType` is already defined in `codingstandards.cpp.Pointers`.

Reply:
Thanks, I'd missed that. However, for this particular case we care about not having specifiers, whereas `PointerOrArrayType` strips the top level specifiers. I've therefore moved this class into `Pointers.qll` as `UnspecifiedPointerOrArrayType` (and deleted some unused code in this file).